glean-core/ios/GleanTests/Metrics/LabeledMetricTests.swift

```diff
@@ -130,17 +130,29 @@ class LabeledMetricTypeTests: XCTestCase {
             subMetric: counterMetric
         )
 
+        // These are fine, now.
         labeledCounterMetric["notSnakeCase"].add(1)
         labeledCounterMetric[""].add(1)
         labeledCounterMetric["with/slash"].add(1)
         labeledCounterMetric["this_string_has_more_than_thirty_characters"].add(1)
 
         XCTAssertEqual(
-            4,
+            0,
             labeledCounterMetric.testGetNumRecordedErrors(.invalidLabel)
         )
 
-        XCTAssertEqual(4, labeledCounterMetric["__other__"].testGetValue())
+        XCTAssertEqual(nil, labeledCounterMetric["__other__"].testGetValue())
+
+        // More than 61 characters? Not okay.
```

Comment:
```suggestion
        // More than 60 characters? Not okay.
```

That's the limit (using `61` is not _wrong_ here, but inaccurate I guess)

Reply:
Ah ha, but excitingly you're wrong! If you put in a 61-character label it will be accepted.

Reply:
then it's wrong in other places!
At least the Python tests say "60 characters" and use 70 and the kotlin tests say "60 characters" and use 62.